Creating a new component using script doesn't work with docker

Type: **minor**

**Describe the bug**

Directions were followed from https://github.com/reactioncommerce/reaction-component-library/blob/master/docs/creating-new-component.md

Running `docker-compose run --rm web node .reaction/scripts/addcomponent MyComponent` from the first step produces the following error:

```
Error: Cannot find module 'fs-extra'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/src/reaction-app/.reaction/scripts/addcomponent.js:1:74)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)```
```

**To Reproduce**

Run `docker-compose run --rm web node .reaction/scripts/addcomponent MyComponent`  at the root of the repo

**Expected behavior**

There should be no error and the component should be created.

**Additional context**

Doing an `npm install` then running `node .reaction/scripts/addcomponent MyComponent` produces the expected result of creating a new component in the proper location.

--- a/docs/creating-new-component.md
+++ b/docs/creating-new-component.md
@@ -19,6 +19,7 @@ Follow this checklist when creating a new component. The overall steps are:
 In the project root directory, run the following command, where `MyComponent` is the name of the component you want to add:
 
 ```bash
+docker-compose run --rm web yarn
 docker-compose run --rm web node .reaction/scripts/addcomponent MyComponent
 ```
 

